README.md

```diff
@@ -1,2 +1,24 @@
 * :speech_balloon: :calling:
 * [docs](https://github.com/DoSomething/gambit/wiki)
+
+# Setup
+## Entire Gambit System w/ Docker
+All Gambit services & required dependencies can be setup locally with docker compose. See the [gambit-services](https://github.com/DoSomething/gambit-services) repo.
+
+## Standalone w/ Docker
+In order to setup this app & required dependencies, simply
+
+1. `git clone`
+2. `docker-compose up`
+
+### Docker setup under the hood
+All apps are executed by `Foreman` to handle process management & mimic Heroku.
+`Nodemon` will autoreload the server when a file changes.
+The compose file defines env variables for connection details & network mapping.
+
+## Standalone without Docker
+1. Run an instance of RabbitMQ, MongoDB. Two options,
```

Comment:
@deadlybutter We're not using RabbitMQ for anything, right?  I'm looking to remove this "Standalone" section since it doesn't seem accurate. 

There's could be argument to remove these changes entirely if you won't be able to help out with support...